src/ucs/stats/stats.c

```diff
@@ -354,15 +355,19 @@ static void ucs_stats_add_to_filter(ucs_stats_node_t *node,
     ucs_list_add_tail(&filter_node->type_list_head, &node->type_list);
     node->filter_node = filter_node;   
 
+    num_counters_out = 0;
     for (i = 0; (i < node->cls->num_counters) && (i < 64); ++i) {
         filter_index = ucs_config_names_search(ucs_global_opts.stats_filter,
                                                node->cls->counter_names[i]);
         if (filter_index >= 0) {
             filter_node->counters_bitmask |= UCS_BIT(i);
+            num_counters_out++;
             found = 1; 
         }
     }
 
+    filter_node->num_counters_filtered = num_counters_out;
```

Comment:
why need this field?
isn't it always popcount of counters_bitmask ?

Reply:
Hi @yosefe , 
This field is required since we need to write it before iterating through the bitmap so it makes sense to store it beforehand,
ucs_stats_serialize_binary() ==>  FWRITE_ONE(&elem->filter_node->num_counters_filtered, stream);

Reply:
why not compute popcount(node->counters_bitmask) and write it?

Reply:
OK, I see - Sounds good, fixed.